app/views/welcome.erb

```diff
@@ -1,11 +1,15 @@
 Welcome to my site!
-
-<% @products.each do |product| %>
-  <h1>
-    <%= "#{product.name} - #{product.price} kr" %>
-  </h1>
-  <p>
-    <%= product.description %>
-  </p>
-
+<% @categories.each do |category| %>
+  <section id="<%= category.titleize.gsub(' ', '').underscore %>">
+    <% @products.each do |product| %>
```

Comment:
You lo-fi states that the category name should be displayed, right? Please add it here.

Reply:
Will do boss!

Reply:
Done! Better?

Reply:
Where did you add this? Can't see it.

Reply:
Now?

Reply:
Done